setup.cfg

```diff
@@ -66,7 +66,7 @@ net =
   tqdm>=4.32.1
   zeep>=3.4.0
 timeseries =
-  cdflib>=0.3.19
+  cdflib>=0.3.19,!=0.4.0
```

Comment:
ooi why? Is it worth a comment?

Reply:
Because it accidentally introduced an `xarray` runtime dep.

Reply:
I put this change in https://github.com/sunpy/sunpy/pull/5911

Reply:
No one can run from xarray.